horizon/opal_relay_api.py

```diff
@@ -122,19 +138,39 @@ async def send_ping(self):
             ),
         ) as response:
             if response.status != status.HTTP_202_ACCEPTED:
-                text = await response.text()
-                raise RuntimeError(
-                    f"Server responded to token request with a bad status {response.status}: {text}"
+                try:
+                    text = await response.text()
+                except:
+                    text = None
+                raise RelayAPIException(
+                    "relay-api",
+                    response.status,
+                    f"Server responded to token request with a bad status {response.status}: {text}",
                 )
         logger.debug("Sent ping.")
 
     async def _run(self):
+        backoff = 0
         while True:
             try:
                 await self.send_ping()
-            except Exception:
-                logger.opt(exception=True).warning("Unable to send ping:")
-            await asyncio.sleep(sidecar_config.PING_INTERVAL)
+                backoff = 0
+            except RelayAPIException as e:
+                logger.warning(
+                    "Could not report uptime status to server: {}. This does not affect the PDP's operational state or data updates.",
```

Comment:
full error should be (this is an example):
Could not report uptime status to server: got HTTP status code 503. This does not affect the PDP's operational state or data updates.

if you keep using the original e.message you can have a much longer message, that is not necessary.